pkg/settings/setting_test.go

```diff
@@ -0,0 +1,26 @@
+package settings
+
+import (
+	"testing"
+)
+
+func TestIsRelease(t *testing.T) {
+	inputs := map[string]bool{
+		"dev":         false,
+		"master-head": false,
+		"master":      false,
+		"v2.5.2":      true,
+		"v2":          true,
+		"v2.0":        true,
+		"v2.x":        true,
+		"v2.5-head":   false,
+		"2.5":         false,
+		"2.5-head":    false,
+	}
+	for key, value := range inputs {
+		result := IsReleaseHelper(key)
+		if result != value {
```

Comment:
You can use `"github.com/stretchr/testify/assert"` here instead

Reply:
Ah good idea.